### ISSUE TITLE
Error on loading addon.

### ISSUE THREAD

**Issue by Aramilgaeda** [Hekili/hekili#533]
**Describe the Bug/Issue**
A console error is occuring when I load or reload the UI. This is happening with the most recent version of Hekili, v9.0.1-1.0.27, however, I have tried rolling back to older versions I still get this error on loading the game or reloading the UI. I saw it said something about Ace3 so I went to CurseForge and installed the Ace3 addon as well, which is version Ace3-Release-r1241, but the error is still happening.

**To Reproduce**
I am using the
/console scriptErrors 1
command.

**Screenshots**

**WHEN SUBMITTING AN ISSUE:**

Please provide the following information.  Leaving it out may result in your ticket being ignored (because I won't have the information needed to explore your report).

1.  The information from the Issue Reporting tab ( /hekili > Issue Reporting ).  This provides your talent and gear information. 
--1-------------------------------
build: v9.0.1-1.0.17
level: 50 (50)
class: ROGUE
spec: assassination

talents: alacrity
    elaborate_planning
    iron_wire
    leeching_poison
    poison_bomb
    subterfuge
    vigor

pvptalents: creeping_venom
   mindnumbing_poison
   smoke_bomb

covenant: none

azerite: azerite_empowered = 3
    echoing_blades = 3
    lying_in_wait = 1
    overwhelming_power = 2
    resounding_protection = 1
    shrouded_suffocation = 1
    twist_the_knife = 1
    unstable_catalyst = 1
    unstable_flames = 1
    vampiric_speed = 1

essences: [blood_of_the_enemy] = 3, breath_of_the_dying = 4, memory_of_lucid_dreams = 3, ripple_in_space = 3

sets/legendaries/artifacts: none

gear: apex_perforator = 1
    argent_crusaders_tabard = 1
    ashjrakamas_shroud_of_resolve = 1
    band_of_the_ancient_dredger = 1
    cephalohide_jacket = 1
    deadeye_spyglass = 1
    desiccators_blessed_gloves = 1
    footpads_of_the_serene_wake = 1
    harlans_loaded_dice = 1
    heart_of_azeroth = 1
    high_altitude_turban = 1
    irradiated_undercoat = 1
    pistoleers_spaulders = 1
    seal_of_questionable_loyalties = 1
    servoarm_bindings = 1
    silvertrimmed_breeches = 1
    trixies_backup_backbiter = 1
    unscrupulous_geologists_belt = 1

legendaries: none

itemIDs: 155881, 158075, 158314, 158353, 158364, 159295, 159297, 159312, 159314, 159331, 159461, 159623, 159725, 168962, 169066, 169223, 169878, 46874

settings: aoe = 3
    buffPadding = 0.25
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = false
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 6
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    gcdSync = true
    maxRefresh = 10
    maxTime = 33
    nameplateRange = 8
    nameplates = true
    package = Assassination
    petbased = false
    potion = potion_of_focused_resolve
    potionsReset = 20180919.1
    throttleRefresh = false
    throttleTime = false
    envenom_pool_pct = 50
    mfd_waste = true
    priority_rotation = false

toggles: cooldowns = true
    custom1 = false
    custom2 = false
    defensives = true
    essences = true
    interrupts = true
    mode = automatic
    potions = false

--1-------------------------------

**PUT ISSUE REPORT LINK HERE:**

2.  If reporting a disagreement with the addon's recommendations, please take a snapshot when you observe the issue and provide the snapshot.  This is not a screenshot.  A snapshot can be generated by binding the Pause feature on the Toggles tab.  When you press Pause, a snapshot of how the addon decided on its CURRENT recommendations is generated and saved to the Snapshots tab.  Copy and paste this to pastebin.com and provide the link here.
 
N/A

3.  If reporting an error (i.e., the addon freezes and/or disables itself), please install the BugSack and BugGrabber addons as they will enable you to paste the exact error message, with detail that will help me identify where the code error is located.  You can paste the error information here or use pastebin.com for that as well.

Message: Interface\AddOns\Hekili\Classes.lua:623: table index is nil
Time: Tue Nov 10 19:16:39 2020
Count: 1
Stack: Interface\AddOns\Hekili\Classes.lua:623: table index is nil
[string "@Interface\AddOns\Hekili\Core.lua"]:185: in function <Interface\AddOns\Hekili\Core.lua:184>
[string "=[C]"]: ?
[string "@Interface\AddOns\Ace3\AceAddon-3.0\AceAddon-3.0.lua"]:70: in function <Interface\AddOns\Ace3\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@Interface\AddOns\Ace3\AceAddon-3.0\AceAddon-3.0.lua"]:527: in function `EnableAddon'
[string "@Interface\AddOns\Ace3\AceAddon-3.0\AceAddon-3.0.lua"]:630: in function <Interface\AddOns\Ace3\AceAddon-3.0\AceAddon-3.0.lua:615>

Locals: <none>

**Comment by Hekili** [comment]
`Interface\AddOns\Hekili\Classes.lua:623: table index is nil`

That doesn't correspond to the current version of the addon.  Use the latest version and post the error message from the latest version.

**Comment by Aramilgaeda** [comment]
Oooff. I had re-downloaded the addon and copied it over to my addons folder and overwrote all the files it said were the same name even before posting this. I couldn't understand how it could be the wrong version when I even checked the file dates of the zip, etc. But since you said it was the wrong version, I just completely deleted the folder from my addons folder, downloads folder, etc. and totally re-downloaded again and it's working now. But I scrolled up again and I can see now how it said "build: v9.0.1-1.0.17" in my info above instead of .27, when I made sure to overwrite everything. Anywho, it's the correct one NOW since I just blew everything away and started over. Thank you for the help!!! :)
